src/meteors/visualize/lime_visualize.py

```diff
@@ -22,6 +22,31 @@
 from meteors.utils.utils import aggregate_by_mask, expand_spectral_mask
 
 
+def _merge_band_names_segments(band_names: dict[str | tuple[str, ...], int]) -> dict[str, int]:
+    """Merges the band names with multiple segments into a single band name.
+
+    Args:
+        band_names (dict[str | tuple[str, ...], int]): A dictionary mapping band names to segment IDs.
+
+    Returns:
+        dict[str, int]: A dictionary mapping band names to segment IDs.
+    """
+
+    # simplify bands with multiple segments
+    segments_to_be_updated = []
+
+    for label in band_names.keys():
+        if isinstance(label, tuple) or isinstance(label, list):
+            new_key = ", ".join(label)
```

Comment:
Not needed the `, ` space after the coma

Reply:
sure, I added it for clarity